From: Lorenzo Bianconi <lorenzo@kernel.org>
Date: Sun, 14 Mar 2021 15:49:19 +0100
Subject: [PATCH] net: export dev_set_threaded symbol

For wireless devices (e.g. mt76 driver) multiple net_devices belongs to
the same wireless phy and the napi object is registered in a dummy
netdevice related to the wireless phy.
Export dev_set_threaded in order to be reused in device drivers enabling
threaded NAPI.

Signed-off-by: Lorenzo Bianconi <lorenzo@kernel.org>
Signed-off-by: David S. Miller <davem@davemloft.net>
---

--- a/net/core/dev.c
+++ b/net/core/dev.c
@@ -6822,6 +6822,7 @@ int dev_set_threaded(struct net_device *
 
 	return err;
 }
+EXPORT_SYMBOL(dev_set_threaded);
 
 void netif_napi_add(struct net_device *dev, struct napi_struct *napi,
 		    int (*poll)(struct napi_struct *, int), int weight)
